pkg/service/schema/service.go

```diff
@@ -116,14 +117,84 @@ func (s Service) CreateSchema(ctx context.Context, request CreateSchemaRequest)
 		jsonSchema[schema.JSONSchemaDescriptionProperty] = request.Description
 	}
 
-	// TODO(gabe) support signing credential schemas
-	// create schema
-	storedSchema := StoredSchema{ID: schemaID, Schema: jsonSchema}
+	// if the schema is a credential schema, the credential's id is a fully qualified URI
+	// if the schema is a JSON schema, the schema's id is a fully qualified URI
+	schemaID := uuid.NewString()
+	schemaURI := strings.Join([]string{s.Config().ServiceEndpoint, schemaID}, "/")
```

Comment:
Actually I think there is something off here. When request is for a `CredentialSchemar2023`, there should be a URL communicated that can be dereferenced to get the credential. We don't have that right now.

Reply:
this should go through the `v1/schemas/id` endpoint though?

Reply:
Create #508 to track